### DISCUSSION TITLE
How do I get this theme? What's it called?

### DISCUSSION THREAD

**Discussion by adriandelgg** [starship/starship#3402] [Q&A] [answered]
Hey everyone, I really like the terminal theme that's in the docs. 

How do I get it?
What's it called?
![theme](https://user-images.githubusercontent.com/32179921/148434554-a9c163fe-6b9f-43d0-b3a9-533d66176347.png)

**Comment by davidkna** [comment, marked as answer]
The demo configuration is listed in the [FAQ section on the website](https://starship.rs/faq/#what-is-the-configuration-used-in-the-demo-gif).
